packages/block-library/src/image/edit.native.js

```diff
@@ -430,6 +433,54 @@ export class ImageEdit extends Component {
 		this.onSetSizeSlug( newValue );
 	}
 
+	onSetFeatured( mediaId ) {
+		const { closeSettingsBottomSheet } = this.props;
+		closeSettingsBottomSheet();
+		setFeaturedImage( mediaId );
+	}
+
+	getSetFeaturedButton( isFeaturedImage ) {
```

Comment:
The get-set in the name `getSetFeaturedButton` is a bit confusing to me. Could it be replaced with `getFeaturedButtonPanel` or similar?

Reply:
Absolutely, I've gone ahead to update the name in https://github.com/WordPress/gutenberg/pull/31415/commits/d17376307ca4f15cc694db95933c35dbb8e3181b. :D